ethpm_types/base.py

```diff
@@ -33,3 +37,31 @@ def json(self, *args, **kwargs) -> str:
             kwargs["exclude_none"] = True
 
         return super().json(*args, **kwargs)
+
+    @classmethod
+    @no_type_check
+    def _get_value(
+        cls,
+        v: Any,
+        to_dict: bool,
```

Comment:
I wonder if you could just use args and kwargs for these? (Except `v` of course)

Reply:
Updated [70ab8e2](https://github.com/ApeWorX/ethpm-types/pull/33/commits/70ab8e2a329bc993570c10809b55645d80a3d384)